myproject/blog/views.py

```diff
@@ -1,6 +1,15 @@
 from django.shortcuts import render
-from django.views.generic import View
+from django.views import View, generic
+from .models import BlogPost
 
 class HomeView(View):
     def get(self, request):
-        return render(request, "blog/home.html")
\ No newline at end of file
+        return render(request, "blog/home.html")
+
+class BlogList(generic.ListView):
+     paginate_by = 5
```

Comment:
Define the settings variable called `DEFAULT_PAGINATED_RECORDS = 5` and use that instead here.

```
paginate_by = settings.DEFAULT_PAGINATED_RECORDS
```